composer.json

```diff
@@ -4,7 +4,8 @@
         "symfony/filesystem": "^4.3",
         "symfony/process": "^4.3",
         "phpseclib/phpseclib": "^3.0.7",
-        "amphp/parallel-functions": "^1.0"
+        "amphp/parallel-functions": "^1.0",
+        "phpseclib/phpseclib": "^3.0.7"
```

Comment:
looks like this line ended up in there twice?

`"phpseclib/phpseclib": "^3.0.7"`

Reply:
Nice catch! Fixed.